experimental/dds/tree2/package.json

```diff
@@ -84,7 +84,6 @@
 	},
 	"devDependencies": {
 		"@fluid-internal/stochastic-test-utils": "workspace:~",
-		"@fluid-internal/test-dds-utils": "workspace:~",
```

Comment:
i think this dep probably got added for fuzz testing at some point. no need to remove it (we're going to want it soon if we don't actually need it already)

Reply:
oops, had a bad view.